PiwikTracker/URLSessionDispatcher.swift

```diff
@@ -94,9 +94,15 @@ final class URLSessionDispatcher: Dispatcher {
 }
 
 fileprivate extension Event {
+    
+    private func cvarParameterValue() -> String {
+        var cvars: Array<String> = customVariables.enumerated().map { "\"\($0.offset + 1)\":[\"\($0.element.name)\",\"\($0.element.value)\"]" }
+        return "{\(cvars.joined(separator: ","))}"
+    }
+    
     var queryItems: [URLQueryItem] {
         get {
-            let items = [
+            var items = [
```

Comment:
Can you keep this as a let and the join al the different query items together in the end?